### PR TITLE
Create user input example

### PR DESCRIPTION
**Pull request by GijsGroote**
find a new class Responder in keyboardInput/keyboard_input_responder
and a example in examples/keyboard_input_example.

Hopefully rebasing went correct.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,7 @@
 urdfEnvs.egg-info/
 *.swp
+
+# ignore intellij config files
+.idea
+idea/
```
Comment (maxspahn) [inline .gitignore:6]: Do we need those ignore items?
It might be related to your local settings.

**Review by maxspahn — CHANGES_REQUESTED** [review]
Looks great. 
Could you check the two items in the ignore list?
See comment.

File: examples/keyboard_input_example.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+from keyboardInput.keyboard_input_responder import Responder
+from pynput.keyboard import Key
+
+
+def main(conn):
+    # copy of examples/tiago.py
+    env = gym.make("tiago-reacher-vel-v0", dt=0.01, render=True)
+    defaultAction = np.zeros(env.n())
+    defaultAction[0:2] = np.array([1.0, 0.0])
+    defaultAction[10] = 0.0
```
Comment (maxspahn) [inline examples/keyboard_input_example.py:15]: To simplify the script, the default action should be removed.

File: examples/keyboard_input_example.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    vel0[5:12] = np.array([0.1, 0.1, 0.2, -0.1, 0.1, 0.2, 0.0])
+
+    for e in range(n_episodes):
+        ob = env.reset(pos=pos0, vel=vel0)
+        print("base: ", ob[0:3])
+        print("torso: ", ob[3])
+        print("head: ", ob[4:6])
+        print("left arm: ", ob[6:13])
+        print("right arm: ", ob[13:20])
+        print("Starting episode")
```
Comment (maxspahn) [inline examples/keyboard_input_example.py:42]: Default printout for tiago is not needed in this example.

File: examples/keyboard_input_example.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    pos0[3] = 0.0
+    # head
+    pos0[4:6] = np.array([1.0, 0.0])
+    # left arm
+    pos0[6:13] = np.array([0.0, 0.0, 0.2, 0.0, 0.0, 0.1, -0.1])
+    # right arm
+    pos0[13:20] = np.array([-0.5, 0.2, 0.2, 0.0, 0.0, 0.1, -0.1])
+    vel0 = np.zeros(19)
+    vel0[0:2] = np.array([0.0, 0.0])
+    vel0[5:12] = np.array([0.1, 0.1, 0.2, -0.1, 0.1, 0.2, 0.0])
```
Comment (maxspahn) [inline examples/keyboard_input_example.py:33]: see above

**Review by maxspahn — COMMENTED** [review]
Added comments on example script.
You should remove all the default action and printout observations.

**Comment by maxspahn** [comment]
Could you also add a section in the readme. So that users are aware of this feature.

File: .gitignore (hunk 1 above)
Comment (GijsGroote) [reply .gitignore:6]: We do not need all the files in the .idea folder which are: 
(venv) gijs@gijs-HP-ZBook-Studio-x360-G5:~/Documents/gym_envs_urdf/.idea$ ls
gym_envs_urdf.iml  inspectionProfiles  misc.xml  modules.xml  other.xml  vcs.xml  workspace.xml

That is why I ignored everything in the .gitignore

I will put this in the .gitignore inside the .idea/ folder. Then I will not appear in the repository.

**Review by maxspahn — CHANGES_REQUESTED** [review]
Everything looks very clean now, except for the binary in [urdfGymExamples/__pycache__/obstacles.cpython-38.pyc](urdfGymExamples/__pycache__/obstacles.cpython-38.pyc).
Could you remove that file.

**Comment by maxspahn** [comment]
I'll wait with merging #23 until this branch is merged.

**Comment by GijsGroote** [comment]
> Everything looks very clean now, except for the binary in [urdfGymExamples/**pycache**/obstacles.cpython-38.pyc](urdfGymExamples/__pycache__/obstacles.cpython-38.pyc). Could you remove that file.

has been removed

**Comment by maxspahn** [comment]
Nice Work @GijsGroote

**Pull request merged**